metacat-common-server/src/main/java/com/netflix/metacat/common/server/services/impl/DatabaseServiceImpl.java

```diff
@@ -11,7 +11,7 @@
  *    limitations under the License.
  */
 
-package com.netflix.metacat.main.services.impl;
+package com.netflix.metacat.common.server.services.impl;
```

Comment:
Implementations should have stayed in main. No reason for them to go in common. Just the interfaces.